acme/acme/crypto_util.py

```diff
@@ -203,7 +203,7 @@ def gen_ss_cert(key, domains, not_before=None,
     """
     assert domains, "Must provide one or more hostnames for the cert."
     cert = OpenSSL.crypto.X509()
-    cert.set_serial_number(1337)
+    cert.set_serial_number(int.from_bytes(OpenSSL.rand.bytes(16),'big'))
```

Comment:
Unfortunately, Python 2 doesn't have `int.from_bytes`, so this can't be used in Python 2-compatible code.